lib/ansible/modules/monitoring/grafana/grafana_dashboard.py

```diff
@@ -79,10 +79,12 @@
       - Override existing dashboard when state is present.
     type: bool
     default: 'no'
-  message:
+  commit_message:
```

Comment:
there are now integration tests for `grafana_dashboard` module. So you should update the `test/integration/targets/grafana_dashboard/tasks` files too.

Reply:
my bad, it's not yet merged : https://github.com/ansible/ansible/pull/64097